modules/core/src/copy.cpp

```diff
@@ -414,6 +414,46 @@ void Mat::copyTo( OutputArray _dst, InputArray _mask ) const
         copymask(ptrs[0], 0, ptrs[2], 0, ptrs[1], 0, sz, &esz);
 }
 
+// just 1 byte is valid
+#define CANT_APPLY_MEMSET int(0xDEADBEEF)
+
+static int check_apply_memset(const Mat *mat, const Scalar &s)
+{
+    int fill_value = CANT_APPLY_MEMSET;
+    switch (mat->depth())
+    {
+    case CV_8U: fill_value = saturate_cast<uchar>( s.val[0] ); break;
+    case CV_8S: fill_value = saturate_cast<schar>( s.val[0] ); break;
+    default:
+        //CV_Error(CV_BadDepth, "Unsupported depth");
```

Comment:
Remove dead code

Reply:
I do not agree.
- for readability : code reader save time without reading unnecessary code below.
- for short-circuit : remove unnecessary operation.